test/common/index.js

```diff
@@ -256,22 +256,21 @@ Object.defineProperty(exports, 'hasFipsCrypto', {
   }
 });
 
-if (exports.isWindows) {
-  exports.PIPE = '\\\\.\\pipe\\libuv-test';
-  if (process.env.TEST_THREAD_ID) {
-    exports.PIPE += `.${process.env.TEST_THREAD_ID}`;
-  }
-} else {
-  exports.PIPE = `${exports.tmpDir}/test.sock`;
+{
```

Comment:
Since the `const` are only used locally and there's a single exported side-effect (`exports.PIPE = ...`) IMHO it's cleaner.
Similar to the `function setupFoo(); setupFoo();` approach in node_bootstap.js
Maybe even V8 will inline the `const`s ?